language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/grpc/helpers/LedgerCallCredentials.java

```diff
@@ -1,13 +1,14 @@
 // Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.ledger.api.auth.client;
+package com.daml.ledger.rxjava.grpc.helpers;
 
 import io.grpc.CallCredentials;
 import io.grpc.Metadata;
 import io.grpc.stub.AbstractStub;
 import java.util.concurrent.Executor;
 
+@SuppressWarnings("deprecation")
```

Comment:
What here is deprecated? Should there be a TODO around here somewhere?

Reply:
It was some fancy instability signalling by the grpc dev team: https://grpc.github.io/grpc-java/javadoc/io/grpc/CallCredentials.html#thisUsesUnstableApi()

Seems ok to remove it now